src/transport.cpp

```diff
@@ -1482,7 +1482,7 @@ SPECIAL(call_elevator)
 {
   int i = 0, j, index = -1;
   long rnum;
-  if (!cmd || !ch || !ch->in_room)
+  if (!cmd || !ch || (!ch->in_room && !ch->in_veh->in_room))
```

Comment:
There's no guarantee that the character is in a vehicle here, and even if this worked you'd still be making it so riggers can't use this remotely. Revert this line, then add a RIG_VEH after it to populate a `struct veh_data *veh` variable (search code for examples). Then you can do vehicle-specific checks with a flow structure like:
```c++
if (veh) {
    if (veh->in_room) {
        do something
    } else {
        send_to_veh some error message
```

Reply:
Actually, thinking about this further, I'm not sure if this roomspec would even trigger at all for someone in a vehicle in the room. This one will be tricky to get to work (maybe trigger the spec function from inside do_push if the only argument is a number or G/B1/B2/etc?)

Reply:
After banging my head on this for several days, I'll toss in the towel. call_elevator() does not work at all for vehicles. I'll leave in the pokestick flag if someone wants to work on this, but I suspect using the mobact logic for elevators is the way to go.